### PR TITLE
WIP: Names list

### PR DESCRIPTION
**Pull request by kagarmoe**
Signed-off-by: kagarmoe <kgarmoe@chef.io>

## Description

It would be nice to know which names are in the code & docs. This WIP list narrows the chef.txt file to obvious names & brands. I removed the obvious IT words and commands.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Chore (non-breaking change that does not add functionality or fix an issue)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have run the pre-merge tests locally and they pass.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] All commits have been signed-off for [the Developer Certificate of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### CODE REVIEW

File: names.txt [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+jaymzh
+jboss
+jkeiser
+jtimberman
+juliandunn
+jwhois
+katello
+klass
+nightlies
+numa
```
Comment (tas50) [inline names.txt:172]: not a name

File: names.txt [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+paulmooring
+queuelen
+slapadd
```
Comment (tas50) [inline names.txt:175]: not a name

File: names.txt [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+slapd
+tsmith
+txqueuelen
+txvlan
+virtuozzo
```
Comment (tas50) [inline names.txt:180]: not a name

File: names.txt [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+katello
+klass
+nightlies
+numa
+paulmooring
+queuelen
+slapadd
+slapd
+tsmith
+txqueuelen
```
Comment (tas50) [inline names.txt:178]: not a name

File: names.txt [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+grantmc
+jaymzh
+jboss
+jkeiser
+jtimberman
+juliandunn
+jwhois
+katello
+klass
+nightlies
```
Comment (tas50) [inline names.txt:171]: not a name

File: names.txt [hunk 6]
```diff
@@ -0,0 +1,182 @@
...
+epel
+grantmc
+jaymzh
+jboss
+jkeiser
+jtimberman
+juliandunn
+jwhois
+katello
+klass
```
Comment (tas50) [inline names.txt:170]: not a name

File: names.txt [hunk 7]
```diff
@@ -0,0 +1,182 @@
...
+clamav
+clearos
+epel
+grantmc
+jaymzh
+jboss
+jkeiser
+jtimberman
+juliandunn
+jwhois
```
Comment (tas50) [inline names.txt:168]: not a name

File: names.txt [hunk 8]
```diff
@@ -0,0 +1,182 @@
...
+Zimmek
+Zpools
+Zygmuntowicz
+advapi
+antergos
+bobberson
+cdoherty
+clamav
+clearos
+epel
```
Comment (tas50) [inline names.txt:161]: not a name

File: names.txt [hunk 9]
```diff
@@ -0,0 +1,182 @@
...
+Splunk
+Stackdriver
+Stratton
+Sutay
+Sysinternals
+Thom
+Timberman
+Toomas
+Vagrantfile
+Validatorless
```
Comment (tas50) [inline names.txt:138]: not a name

File: names.txt [hunk 10]
```diff
@@ -0,0 +1,182 @@
...
+Sölvi
+Splunk
+Stackdriver
+Stratton
+Sutay
+Sysinternals
+Thom
+Timberman
+Toomas
+Vagrantfile
```
Comment (tas50) [inline names.txt:137]: not a name

File: names.txt [hunk 11]
```diff
@@ -0,0 +1,182 @@
...
+Rockwood
+Rubo
+Rubygems
+Salim
+Sangoma
+Sauve
+Scaleway
+Schuberg
+Serdar
+Serverspec
```
Comment (tas50) [inline names.txt:120]: not a name

File: names.txt [hunk 12]
```diff
@@ -0,0 +1,182 @@
...
+Prajakta
+Purohit
+Rakefile
+Rakvåg
+Ranjib
+Raspbian
+Redistributable
+Renatus
+Rockwood
+Rubo
```
Comment (tas50) [inline names.txt:112]: not a name

File: names.txt [hunk 13]
```diff
@@ -0,0 +1,182 @@
...
+Mesos
+Mundrawala
+Munki
+Murawski
+Mzyk
+Nagios
+Nehate
+Netplan
+Nimisha
+Nuget
```
Comment (tas50) [inline names.txt:91]: not a name

File: names.txt [hunk 14]
```diff
@@ -0,0 +1,182 @@
...
+Kubb
+Kubernetes
+Lellan
+Lockdown
+Macports
+Magnus
+Maier
+Makefiles
+Mangeia
+Manjaro
```
Comment (tas50) [inline names.txt:79]: not a name

File: names.txt [hunk 15]
```diff
@@ -0,0 +1,182 @@
...
+Kirill
+Korn
+Kubb
+Kubernetes
+Lellan
+Lockdown
+Macports
+Magnus
+Maier
+Makefiles
```
Comment (tas50) [inline names.txt:77]: not a name

File: names.txt [hunk 16]
```diff
@@ -0,0 +1,182 @@
...
+Kaustubh
+Keiser
+Khatri
+Kirill
+Korn
+Kubb
+Kubernetes
+Lellan
+Lockdown
+Macports
```
Comment (tas50) [inline names.txt:74]: not a name

File: names.txt [hunk 17]
```diff
@@ -0,0 +1,182 @@
...
+Dokken
+Dubey
+Eachern
+Erchef
+Erubis
+Etcd
+Fauxhai
+Foodcritic
+Forwardable
+GRPC
```
Comment (tas50) [inline names.txt:44]: not a name

File: names.txt [hunk 18]
```diff
@@ -0,0 +1,182 @@
...
+Botman
+Brightcove
+Brimager
+Buildkite
+Cavalca
+Chisamore
+Chouhan
+Cloke
+Cloudera
+Cmdlet
```
Comment (tas50) [inline names.txt:22]: not a name

File: names.txt [hunk 19]
```diff
@@ -0,0 +1,182 @@
...
+Argumentlist
+Artifactory
+Ashwini
+Awasthi
+Azurerm
+Bento
+Berksfile
+Berksfiles
+Berkshelf
+Bigtable
```
Comment (tas50) [inline names.txt:12]: not a name

File: names.txt [hunk 20]
```diff
@@ -0,0 +1,182 @@
+Alibaba
+Antima
+Argumentlist
+Artifactory
+Ashwini
+Awasthi
+Azurerm
```
Comment (tas50) [inline names.txt:7]: not a name

**Comment by kagarmoe** [comment]
Cool! Thanks!

File: names.txt (hunk 20 above)
Comment (kagarmoe) [reply names.txt:7]: I double-dog-dare you.

**Pull request closed without merging**